src/main/java/ro/msg/learning/bank/controllers/AccountController.java

```diff
@@ -0,0 +1,48 @@
+package ro.msg.learning.bank.controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.bank.dtos.AccountDTO;
+
+import ro.msg.learning.bank.services.AccountService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class AccountController {
+private final AccountService accountService;
+
+
+    @GetMapping("/accounts")
+    public List<AccountDTO> listAll() {
+        return accountService.listAll();
+    }
+
+    @GetMapping("/accounts/{iban}")
+    public AccountDTO findByIban(@PathVariable int iban) {
+        return accountService.findByIban(iban);
+    }
+
+    @PostMapping("/accounts")
+    public AccountDTO create(@RequestBody AccountDTO accountDTO) {
+        return accountService.create(accountDTO);
+    }
+
+
+    @DeleteMapping("/accounts/{iban}")
+    public void deleteById(@PathVariable int iban) {
```

Comment:
Usually you return back the deleted object for the following possible reason: In some cases it is needed to display some information in the UI regarding the deleted object, such as a message: "Saving Account (this is the type) from Banca Transilvania (this is the bank) was deleted.